jujugui/static/gui/src/app/components/deployment-bar-notification/deployment-bar-notification.js

```diff
@@ -0,0 +1,99 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2015 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+YUI.add('deployment-bar-notification', function() {
+
+  juju.components.DeploymentBarNotification = React.createClass({
+    timeout: null,
+
+    /**
+      Fade out the notification.
+
+      @method _hideNotification
+    */
+    _hideNotification: function() {
+      var node;
+      try {
+        node = React.findDOMNode(this);
+      } catch (e) {
+        // The notification has already been removed.
+      }
+      // Fade out the notification.
+      if (node) {
+        node.classList.remove('deployment-bar__notification--visible');
```

Comment:
this can be moved into the `try` block and then you can save on the conditional.